src/lib/dbHelper.ts

```diff
@@ -1,26 +1,33 @@
+import * as squel from 'squel'
+
 interface Field {
   name: string
   type: string
   options: string[]
 }
 
-export interface TableOptions {
-  name: string
-  fields: Field[]
-}
-
-interface DerivedKeyOptions {
+export interface DerivedKeyOptions {
```

Comment:
I would name it `Attributes`, options is a bit confusing - the same in all other interfaces